.changeset/twelve-walls-deny.md

```diff
@@ -0,0 +1,7 @@
+---
+"@wagmi/connectors": minor
```

Comment:
When the Changesets version PR is created, double check that `@wagmi/connectors` doesn't get bumped to `5.0.0` (has in the past, need to take a look at the Changesets config to figure out why).